dubbo-config/dubbo-config-spring/src/main/resources/META-INF/compat/dubbo.xsd

```diff
@@ -1,12 +1,12 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <xsd:schema xmlns:xsd="http://www.w3.org/2001/XMLSchema"
-            xmlns:beans="http://www.springframework.org/schema/beans"
+            xmlns:beans="http://www.springframework.org/schema/beans" 
             xmlns:tool="http://www.springframework.org/schema/tool"
             xmlns="http://code.alibabatech.com/schema/dubbo"
             targetNamespace="http://code.alibabatech.com/schema/dubbo">
 
     <xsd:import namespace="http://www.w3.org/XML/1998/namespace"/>
-    <xsd:import namespace="http://www.springframework.org/schema/beans"/>
+    <xsd:import namespace="http://www.springframework.org/schema/beans" schemaLocation="http://www.springframework.org/schema/beans/spring-beans-4.3.xsd"/>
```

Comment:
Please change `http://www.springframework.org/schema/beans/spring-beans-4.3.xsd` to `http://www.springframework.org/schema/beans/spring-beans.xsd`